src/http_server.cc

```diff
@@ -2813,6 +2813,18 @@ HTTPAPIServer::InferRequestClass::InferRequestClass(
   evhtp_connection_t* htpconn = evhtp_request_get_connection(req);
   thread_ = htpconn->thread;
   evhtp_request_pause(req);
+
+  auto request_type = GetEnvironmentVariableOrDefault("RequestType", "DEFAULT");
```

Comment:
should be response type? use an env like AB_RESPONSE_TYPE to have the same style of our other env vars.